pkg/internal/apis/config/validate_test.go

```diff
@@ -428,3 +431,69 @@ func TestPortValidate(t *testing.T) {
 		})
 	}
 }
+
+func TestValidatePortMapping(t *testing.T) {
+	portMappingsSet := sets.NewString()
+	addSet := sets.NewString()
+
+	newPortMapping := func(addr string, port int, protocol string) PortMapping {
+		return PortMapping{
+			HostPort:      int32(port),
+			ListenAddress: addr,
+			Protocol:      PortMappingProtocol(protocol),
+		}
+	}
+	errMsg := "port mapping with same listen address, port and protocol already configured"
+	cases := []struct {
+		mapping   PortMapping
```

Comment:
add a name field and use named variables , take previous test as an example, is easier to read and go through the test cases